modules/velero-obtaining-by-downloading.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-downloading_{context}"]
+= Downloading the Velero CLI tool
+
+You can download and install the Velero CLI tool by following the instructions on the link:https://velero.io/docs/v1.8/basic-install/#install-the-cli[Veleo documentation page].
+
+The page includes instructions for:
+
+* MacOS using Homebrew
+* GitHub
+* Windows using Chocolatey
+
+.Prerequisites
+
+* Access to a Kubernetes cluster, v1.16 or later, with DNS and container networking enabled.
+* `kubectl` installed locally.
+
+.Procedure
+
+. Open a browser and navigate to link:https://velero.io/docs/v1.8/basic-install/#install-the-cli["Install the CLI" on the Verleo website].
+. Follow the appropriate procedure for MacOS, GitHub, or Windows.
+. If possible, download the Velero version appropriate for your version of OADP, according to the table that follows:
+
+[cols="2", options="header"]
+.OADP-Velero version relationship
```

Comment:
You'll want to `+` this under the final list item so that it doesn't look like this: 
![image](https://user-images.githubusercontent.com/50179998/169885605-c6ee13d4-0847-4a32-8ecd-2e9248c94c30.png)

Reply:
I'd also note that the convention is to structure titled table markup like so:

```
.Table title
[table, options]
|====
...
|====
```